app/views/admin/slack_presence_checks/index.html.erb

```diff
@@ -13,5 +13,7 @@
       <td><%= check.presence %></td>
     </tr>
   <% end %>
+
+  <%= paginate @checks %>
```

Comment:
It looks like this is generating routes with uris like `/slack_presence_checks/page/1`. I think it would be best to have them generate as query params like `/slack_presence_checks?page=1`. Then you wouldn't have to add any routes and the rest of your code should still work

Reply:
I think this is completed now, i'm not 100% sure, it passes my tests, but I would appreciate a second look, thanks!

Reply:
Yep, looks like it's working!